src/api/db/structure.sql

```diff
@@ -281,7 +281,7 @@ CREATE TABLE `bs_requests` (
 
 CREATE TABLE `cache_lines` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `key` varchar(4096) COLLATE utf8_unicode_ci NOT NULL,
+  `key` varchar(255) COLLATE utf8_unicode_ci NOT NULL,
```

Comment:
Take care, this can introduce a regression here, was you who changed it before to 4096 IIRC